toonz/sources/toonz/comboviewerpane.cpp

```diff
@@ -737,14 +714,6 @@ void ComboViewerPanel::onSceneChanged() {
   int frameIndex    = fh->getFrameIndex();
   int maxFrameIndex = fh->getMaxFrameIndex();
   if (frameIndex > maxFrameIndex) maxFrameIndex = frameIndex;
-
```

Comment:
I posted this in the other PR, but I think having it on scene changed is actually useful and less confusing for users.  If a user changes the scene fps in the scene settings, but the viewer doesn't update, I think that would confuse users.